content/FAQ.md

```diff
@@ -46,6 +46,43 @@ There are several ways to get involved.
 #### Can KEDA be used in production?
 Yes! KEDA is now 1.0 and suited for production workloads.
 
+#### How to make KEDA work in a proxy network?
+If while setting up KEDA, you get an error: "(v1beta1.external.metrics.k8s.io) status FailedDiscoveryCheck" with a Message: no response from https://ip:443: Get https://ip:443: net/http: request canceled while waiting for connection (Client.Timeout exceeded while awaiting headers); one of the reason for this can be that you are behind a proxy network.
```

Comment:
```suggestion
If while setting up KEDA, you get an error: `(v1beta1.external.metrics.k8s.io) status FailedDiscoveryCheck` with a message: `no response from https://ip:443: Get https://ip:443: net/http: request canceled while waiting for connection (Client.Timeout exceeded while awaiting headers)`.

 One of the reason for this can be that you are behind a proxy network.
```